### DISCUSSION TITLE
use both self host and public ID server

### DISCUSSION THREAD

**Discussion by aveerlapati** [rustdesk/rustdesk#9041] [Q&A]
Hi

how do you use both self host and Rustdesk public ID servers at the same time?

sometimes when my ISP who is providing public IP goes down automatically my backup ISP who doesn't provide public IP comes online

this case it will be helpful to connect my devices using Rustdesk public ID sever when my ISP public IP goes offline

thank you very much!

**Comment by rustdesk** [comment]
a lot of duplciate disucssions
